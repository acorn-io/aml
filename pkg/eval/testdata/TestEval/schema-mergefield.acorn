schema {
		foo: string
		foo: string == "hi"
}

foo: "hi"
