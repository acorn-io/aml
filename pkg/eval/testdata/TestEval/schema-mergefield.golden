{
  "Foo": {
    "type": "object",
    "properties": {
      "foo": {
        "type": "string"
      }
    }
  },
  "foo": "hi"
}